### ISSUE TITLE
why my test acc is so low

### ISSUE THREAD

**Issue by Yuzhang-Peng** [Lornatang/pytorch-vgg19-cifar100#1]
My training set accuracy rate is 90+, but the test set is less than 60.

**Comment by AlloyMei** [comment]
me too.
